RabbitMQ.pm

```diff
@@ -275,6 +293,12 @@ C<$props> is an optional hash (the AMQP 'props') respecting the following keys:
 
 =head2 consume($channel, $queuename, $options)
 
+Notify the RabbitMQ server that messages for the specified queue should
+be delivered on the specified channel.
```

Comment:
This should probably just say that it puts a channel into consume mode. You don't actually tell the server anything other than that, really. You can enter consume mode and do nothing, and then die, and the server won't have done anything but ack your RPC calls.